prediction_market_agent/agents/microchain_agent/microchain_agent.py

```diff
@@ -25,6 +25,7 @@
 {engine_help}
 
 Only output valid Python function calls.
+Make 'Reasoning' calls frequently - at least every other call.
```

Comment:
Instead of just making `Buy...`, `Sell...`,`Predict...` calls in succession, it intersperses these with reasoning. Now more 'chain-of-thought'-y